apps/server/src/modules/files-storage/index.ts

```diff
@@ -1,5 +1,5 @@
+export * from './files-preview-amqp.module';
 export * from './files-storage-amqp.module';
 export * from './files-storage-api.module';
-export * from './files-storage-test.module'; // @deprecated remove after move api tests to modules
```

Comment:
Warum hast du das Test Modul hier gelöscht? Es wird aktuell noch in den API Tests verwendet.

Reply:
Weil die sind ins modul verschoben wurden. Das heist nach außen nicht mehr exportieren